lib/cucumber/formatter/fail_fast.rb

```diff
@@ -9,7 +9,7 @@ class FailFast
       def initialize(configuration)
         configuration.on_event :test_case_finished do |event|
           _test_case, result = *event.attributes
-          Cucumber.wants_to_quit = true unless result.ok?(configuration.strict)
+          Cucumber.wants_to_quit = !result.ok?(configuration.strict)
```

Comment:
Since `Cucumber.wants_to_quit` is global and may be set to `true` by other components than the `FailFast` formatter, I think that it should only set `Cucumber.wants_to_quit` to false when it is sure that it is a flaky test that have passed.